development/openstack-base-focal-wallaby/README.md

```diff
@@ -1,269 +1,371 @@
+    openstack router add subnet provider-router int_subnet
+
+See the [Neutron documentation][openstack-neutron] for more information.
+
+## Create a flavor
 
-    openstack router add subnet provider-router internal_subnet
+Create at least one flavor to define a hardware profile for new instances. Here
+we create one called 'm1.tiny':
 
-Neutron provides a wide range of configuration options; see the [OpenStack Neutron][] documentation for more details.
+    openstack flavor create --ram 1024 --disk 10 --ephemeral 10 m1.tiny
 
-### Configuring a flavor
+Make sure that your MAAS nodes can accommodate the flavor's resources.
 
-Starting with the OpenStack Newton release, default flavors are no longer created at install time. You therefore need to create at least one machine type before you can boot an instance:
+## Import an SSH keypair
 
-    openstack flavor create --ram 2048 --disk 20 --ephemeral 20 m1.small
+An SSH keypair needs to be imported into the cloud in order to access your
+instances.
 
-### Booting an instance
+Generate one first if you do not yet have one. This command creates a
+passphraseless keypair (remove the `-N` option to avoid that):
 
-First generate an SSH keypair so that you can access your instances once you've
-booted them:
+    ssh-keygen -q -N '' -f ~/cloud-keys/id_mykey
 
-    mkdir -p ~/.ssh
-    touch ~/.ssh/id_rsa_cloud
-    chmod 600 ~/.ssh/id_rsa_cloud
-    openstack keypair create mykey > ~/.ssh/id_rsa_cloud
+To import a keypair:
 
-> **Note**: you can also upload an existing public key to the cloud rather than
-  generating a new one:
+    openstack keypair create --public-key ~/cloud-keys/id_mykey.pub mykey
 
-    openstack keypair create --public-key ~/.ssh/id_rsa.pub mykey
+## Configure security groups
 
-You can now boot an instance on your cloud:
+To allow ICMP (ping) and SSH traffic to flow to cloud instances create
+corresponding rules for each existing security group:
 
-    openstack server create --image focal --flavor m1.small --key-name mykey \
-        --network internal focal-test
+    for i in $(openstack security group list | awk '/default/{ print $2 }'); do
+       openstack security group rule create $i --protocol icmp --remote-ip 0.0.0.0/0;
+       openstack security group rule create $i --protocol tcp --remote-ip 0.0.0.0/0 --dst-port 22;
+    done
 
-### Attaching a volume
+You only need to perform this step once.
 
-First, create a 10G volume in cinder:
+## Create an instance
 
-    openstack volume create --size=10 <name-of-volume>
+Create a Focal amd64 instance called 'focal-1':
 
-then attach it to the instance we just booted:
+    openstack server create --image focal-amd64 --flavor m1.tiny \
+       --key-name mykey --network int_net \
+        focal-1
 
-    openstack server add volume focal-test <name-of-volume>
+## Assign a floating IP address
 
-The attached volume will be accessible once you login to the instance (see below).  It will need to be formatted and mounted!
```

Comment:
I know why you removed the two sections. One might argue the volume one should stay as this is analogous to the floating IP. But I won't die on that hill.

Reply:
I'd like to leave them out.